package.json

```diff
@@ -8,7 +8,7 @@
     "build": "babel src -d lib --ignore specs.js,*.md --copy-files",
     "ci": "npm run lint && npm test",
     "lint": "eslint src",
-    "precommit": "lint-staged && ./rebuild-docs.sh",
```

Comment:
Apparently we need this for windows users. Should we replace this with `sh rebuild-docs.sh` instead for cross compatibility?

Reply:
Done.

Reply:
No issues with SourceTree?

Reply:
Nope, worked fine.